### PR TITLE
New UI Instructions Screen

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: lib/src/androidTest/java/com/smileidentity/compose/selfie/SelfieCaptureScreenTest.kt [hunk 1]
```diff
@@ -128,21 +119,22 @@ class SelfieCaptureScreenTest {
...
-        composeTestRule.apply {
-            setContent { SelfieCaptureScreen(viewModel = viewModel) }
-            waitUntilAtLeastOneExists(hasTestTag(takePictureTag))
-        }
-
-        // then
-        verify(atLeast = 1, timeout = 1000) { viewModel.analyzeImage(any(), camSelector) }
-    }
+    // todo broke test
+    // @OptIn(ExperimentalTestApi::class)
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply lib/src/androidTest/java/com/smileidentity/compose/selfie/SelfieCaptureScreenTest.kt:123]: yes

File: sample/src/main/java/com/smileidentity/sample/compose/Theme.kt [hunk 2]
```diff
@@ -22,7 +23,7 @@ fun SmileIDTheme(content: @Composable () -> Unit) {
 
     MaterialTheme(
         colorScheme = SmileID.colorScheme,
-        typography = SmileID.typography,
+        typography = SmileID.typographyv2,
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply sample/src/main/java/com/smileidentity/sample/compose/Theme.kt:26]: I am doing this on the new screens to experiment new theming - and, this feature does not get to main branch
